src/index.js

```diff
@@ -46,11 +46,23 @@ class Radar {
     return STATUS;
   }
 
-  static initialize(publishableKey) {
+  static initialize(publishableKey, options={}) {
     if (!publishableKey) {
       console.error('Radar "initialize" was called without a publishable key');
     }
     Storage.setItem(Storage.PUBLISHABLE_KEY, publishableKey);
+
+
+    const {cacheLocationMinutes} = options
```

Comment:
nit: space around the brackets `{ cacheLocationMinutes }` for consistency, and semicolon at end

Reply:
👍